velox/exec/MemoryReclaimer.h

```diff
@@ -38,6 +38,28 @@ class MemoryReclaimer : public memory::MemoryReclaimer {
   MemoryReclaimer() = default;
 };
 
+/// Provides the default node memory reclaimer implementation for velox task
+/// execution. It parallelize the child memory pool reclamation from the same
+/// node memory pool.
+class NodeMemoryReclaimer : public memory::MemoryReclaimer {
```

Comment:
Can we put this in Task.h?

Reply:
It is better to put it here with the default leaf level memory reclaimer which is independent of task.